src/tla2tools.ts

```diff
@@ -25,10 +25,12 @@ const NO_ERROR = 0;
 const MIN_TLA_ERROR = 10;           // Exit codes not related to tooling start from this number
 const LOWEST_JAVA_VERSION = 8;
 const DEFAULT_GC_OPTION = '-XX:+UseParallelGC';
+const TLA_CMODS_LIB_NAME = 'CommunityModules-deps.jar';
```

Comment:
TLC automatically side-loads `CommunityModules-deps.jar` if present.

https://github.com/tlaplus/tlaplus/blob/33bc082a84af2190cae3f6775bb08891b81eba38/tlatools/org.lamport.tlatools/customBuild.xml#L422

Reply:
I see. This works for Java if I understand correctly, but for the TLAPS to use modules from both jars, I have to provide them both as module search paths.

By the way, I added support for using modules from zips/jars in TLAPM.